src/core/connection.c

```diff
@@ -2243,7 +2244,9 @@ QuicConnGenerateLocalTransportParameters(
     LocalTP->MaxUdpPayloadSize =
         MaxUdpPayloadSizeFromMTU(
             CxPlatSocketGetLocalMtu(
-                Connection->Paths[0].Binding->Socket));
+                Connection->Paths[0].Binding->Socket,
+                Connection->Paths[0].Route.UseQTIP
+            ));
```

Comment:
Let's pass the `Route` itself, and make some minor changes to style.
```suggestion
                Connection->Paths[0].Binding->Socket,
                &Connection->Paths[0].Route));
```

Reply:
Fixed.